packages/flutter/lib/src/material/slider.dart

```diff
@@ -1064,7 +1063,7 @@ class _RenderSlider extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
       curve: Curves.fastOutSlowIn,
     )..addStatusListener((AnimationStatus status) {
       if (status == AnimationStatus.dismissed && _state.overlayEntry != null) {
-        _state.overlayEntry!.remove();
```

Comment:
The original code seems fine to me, I think if this become null somehow. Some thing may be broken and we would want to know